packages/trader/src/Modules/Trading/Components/Form/TradeParams/Turbos/payout-selector.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { observer } from '@deriv/stores';
+import { useTraderStore } from 'Stores/useTraderStores';
+import { useDevice } from '@deriv-com/ui';
+import PayoutPerPointInput from 'Modules/Trading/Components/Elements/PayoutPerPoint/payout-per-point-input';
+import { LabelPairedChevronsDownCaptionRegularIcon, LabelPairedChevronsUpCaptionRegularIcon } from '@deriv/quill-icons';
+import './payout-selector.scss';
+
+const PayoutSelector = observer(() => {
+    const { barrier_1, payout_choices, setPayoutPerPoint, togglePayoutWheelPicker, payout_per_point, currency } =
+        useTraderStore();
+    const { isMobile } = useDevice();
+
+    const toggleBarriersTable = () => {
+        togglePayoutWheelPicker();
+    };
+
+    const onPayoutClick = (value: string) => {
+        setPayoutPerPoint(value);
```

Comment:
Same here, can we pass the value directly to setPayoutPerPoint instead of onPayoutClick?